habitat-lab/habitat/tasks/rearrange/sub_tasks/cat_nav_to_obj_sensors.py

```diff
@@ -82,7 +82,7 @@ def get_observation(
         pan_obs = observations["robot_head_panoptic"]
         max_obs_val = np.max(pan_obs)
         obs = np.zeros(
-            (self._dimensionality, self._dimensionality, self._num_channels),
+            (pan_obs.shape[0], pan_obs.shape[1], self._num_channels),
```

Comment:
I think the observation space will have to change accordingly

Reply:
Yes I am fixing this properly for all sensors

Reply:
Done.